tasks/uk/census/wrapper.py

```diff
@@ -0,0 +1,132 @@
+from collections import OrderedDict, defaultdict
+import itertools
+
+from tasks.meta import current_session
+from tasks.util import MetaWrapper, TableTask
+from tasks.uk.cdrc import OutputAreas, OutputAreaColumns
+from tasks.uk.census.metadata import CensusColumns
+
+from .ons import ImportUK, ImportEnglandWalesLocal
+from .scotland import ImportScotland
+from .metadata import parse_table, COLUMNS_DEFINITION
+
+
+class Census(TableTask):
+    # Which task to use to import an specific table
+    REGION_MAPPING = {
+        "UK": ImportUK,
+        "EW": ImportEnglandWalesLocal,
+        "SC": ImportScotland
+    }
+
+    def requires(self):
+        deps = {
+            'geom_columns': OutputAreaColumns(),
+            'data_columns': CensusColumns(),
+        }
+        for t in self.source_tables():
+            deps[t] = self.task_for_table(t)
+        return deps
+
+    def timespan(self):
+        return 2011
+
+    def columns(self):
+        cols = OrderedDict()
+        input_ = self.input()
+        cols['GeographyCode'] = input_['geom_columns']['oa_sa']
+        cols.update(input_['data_columns'])
+        return cols
+
+    def source_tables(self):
+        return set(itertools.chain(*[[d['table'] for d in col['data']] for col in COLUMNS_DEFINITION.values()]))
+
+    def task_for_table(self, table):
+        _, region = parse_table(table)
+        return self.REGION_MAPPING[region](table=table)
+
+    def populate(self):
+        '''
+        Joins data from all UK sub-census dependencies.
+
+        For each column, there are multiple possiblities:
+          - The column has a single datasource and field: ``"data" : [{"table": "t", "fields": ["f"]}]``.
+            The table will be added to the FROM, and the column to the SELECT
+            ``SELECT t.f AS id FROM ... FULL JOIN t USING (geographycode)``
```

Comment:
For a better understanding I'd change `id` for `out_data` or something like that